lib/cookie.c

```diff
@@ -1144,7 +1144,7 @@ struct CookieInfo *Curl_cookie_init(struct Curl_easy *data,
     while(Curl_get_line(line, MAX_COOKIE_LINE, fp)) {
       if(checkprefix("Set-Cookie:", line)) {
         /* This is a cookie line, get it! */
-        lineptr = &line[11];
+        lineptr = &line[ strlen("Set-Cookie:") ];
```

Comment:
Can create runtime overhead

Reply:
no, it can't https://godbolt.org/z/G6orTf
try to change string and you'll see what changed in resulted asm.
this is a basic optimization

Reply:
Yes, it can: https://godbolt.org/z/Gx68c9
The optimization is not in any standard, especially not in the target C specification, so one cannot rely on it.

Reply:
>  this is a basic optimization

C has no optimization AFAIK